komodo/data/activator_switch.tmpl

```diff
@@ -1,10 +1,32 @@
+{% if py_version == "py38" -%}
 if [[ $(uname -r) == *el7* ]] ; then
     export KOMODO_ROOT={{ prefix }}
     KOMODO_RELEASE_REAL={{ release }}
 
     source $KOMODO_ROOT/$KOMODO_RELEASE_REAL-rhel7/enable
     export PS1="(${KOMODO_RELEASE_REAL}) ${_PRE_KOMODO_PS1}"
     export KOMODO_RELEASE=$KOMODO_RELEASE_REAL
-else
+elif [[ $(uname -r) == *el8* ]] ; then
+    export KOMODO_ROOT={{ prefix }}
+    KOMODO_RELEASE_REAL={{ release }}
+
+    source $KOMODO_ROOT/$KOMODO_RELEASE_REAL-rhel8/enable
+    export PS1="(${KOMODO_RELEASE_REAL}) ${_PRE_KOMODO_PS1}"
+    export KOMODO_RELEASE=$KOMODO_RELEASE_REAL
+{%- else -%}
+if [[ $(uname -r) == *el7* ]] ; then
+    echo -e "Warning! Komodo release for python3.11 is not available on RHEL7."
```

Comment:
should we write for "python newer than 3.8" to be in sync with what the tests actually tests?

Reply:
Yup, this is an error, not warning.